en/guide/mavlink_router.md

```diff
@@ -0,0 +1,276 @@
+# MAVLink Router
+
+The [mavlink-router](https://github.com/mavlink-router/mavlink-router) can be used to route and filter MAVLink packets between UART, UDP and TCP endpoints, and to log MAVLink traffic.
+
+This topic provides an overview of what the router can do, and how it is used.
+See the [official docs](https://github.com/mavlink-router/mavlink-router) for more information, including instructions for building and installation.
+
+
+## Using mavlink-router
+
+MAVLink router can be started in a terminal using the `mavlink-routerd` command.
+The endpoints and other options can be configured using [command line arguments](#command-line-syntax) or a [configuration file](#configuration-file).
+
+A simple usage is to specify a single endpoint using the `e` flag, and then the routed interface: 
+
+```bash
+mavlink-routerd -e 10.73.41.30:14550 127.0.0.1:14550
+```
+
+A number of examples are provided below
+
+
+### Routing from a UART to other endpoints
+
+To route mavlink packets from `ttyS1` to two UDP endpoints (from the official docs!):
+
+```bash
+mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550 /dev/ttyS1:1500000
+```
+
+Note that the `1500000` after the colon in `/dev/ttyS1:1500000` sets the UART baudrate.
+
+It's also possible to route mavlinks packets from any interface using:
+
+```bash
+mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550  0.0.0.0:24550
+```
+
+`mavlink-router` also listens, by default, on port 5760 for TCP connections. 
+Any connection there will also receive routed packets.
+
+IPv6 addresses must be enclosed in square brackets like this: `[::1]`.
+The port number can be specified in the same way, as with IPv4 then: `[::1]:14550`.
+
+
```

Comment:
maybe we should also mention the provided systemd service. Provided one installed via package or via `ninja -C build install`, the normal `systemctl <start|enable> mavlink-router` can be used to make it permanent (which makes sense when it's running on a companion computer) rather than starting it manually or via hack scripts.